scrowl-io/scrowl
# Sc 1379 delete term from glossary

### Short summary

1. Adds delete functionality and 2. fixes bug on ActionMenu

The additional request for a larger input on the description field will require an Owl-UI update to allow for textareas. For now I've just applied a height style to approximate the design.

---

### Test steps

1. User should be able to click the dropdown menu beside a glossary term, then click Delete to 'soft' remove the term from the glossary list. 
1a. Navigate to Home/Preferences, then navigate back--deleted term should be back in the last
1b. Save the project (from the application menu or CMD+S). Navigate to Home/Preferences, then navigate back--deleted term should remain deleted
3. User should be able to click any part of the ActionMenu item (icon, text, div area) to trigger its respective action
